README.md

```diff
@@ -4,7 +4,7 @@
 <h3><a href="http://humanisticrobotics.com/products/safe-remote-control" target="_blank">Safe Remote Control System (SRCS)</a></h3>
 
 
-#### Tutorial Instructions: [Running_Tutorials] (Running_Tutorials.md)
-#### C Drivers: [/drivers] (/drivers)
-#### ROS Drivers: [/hri_safety_sense] (/hri_safety_sense)
+#### Tutorial Instructions: [Running_Tutorials](Running_Tutorials.md)
+#### C Drivers: [/drivers](/drivers)
```

Comment:
```suggestion
#### C Drivers: [drivers](drivers/README.md)
```

You'll also have to write a README.md for the drivers subdirectory.

Reply:
Looks like `Running_Tutorials.md` is specifically for the 4 tutorials in the C driver. Should we just move that into `hri_c_driver` and use that as the README for that package, then just have 2 links instead of 3 links in the top-level README? As far as I can tell, the tutorials are the only way to run the C driver anyway, because the `.so` file doesn't run on its own, it just seg faults.

Reply:
Yes, that sounds good, let's do that.

Reply:
Done in fc1be8f